lib/vrf.c

```diff
@@ -1032,3 +1032,18 @@ vrf_id_t vrf_generate_id(void)
 
 	return ++vrf_id_local;
 }
+
+vrf_id_t vrf_to_id(struct vrf *vrf)
```

Comment:
Aren't the additions of the accessor functions and the conversion over to them a seperate logical thing then what this commit is doing?  This should be broken out.

Reply:
what I can do is to make available that macro before the application of the change of the commit.
that is to say that there are too many occurences to change.